build/squire-raw.js

```diff
@@ -2292,6 +2292,50 @@ var onPaste = function ( event ) {
         }
     }, 0 );
 };
+
+var onDrop = function( event ) {
+    var dataTransfer = event.dataTransfer,
+        types = dataTransfer && dataTransfer.types;
+
+    var hasFiles = ( types && ( indexOf.call( types, 'Files' ) >= 0 ));
+
+    if( !hasFiles ) {
+        var self = this;
+
+        var insertHtmlItem = function ( html ) {
+            self.insertHTML( html, true );
+        };
+
+        if( dataTransfer.items ) {
+            for( var i = 0; i < dataTransfer.items.length; i++ ) {
+                var item = dataTransfer.items[i];
+                if( item.type === 'text/html') {
+                    event.preventDefault();
+
+                    item.getAsString( insertHtmlItem );
```

Comment:
Chrome 50 has a bug in getAsString()  when pasting content (HTML) from some MS applications. Do you see extra characters when you drag from Outlook or MS Word?